src/sourcemaps/tools/rollup.ts

```diff
@@ -0,0 +1,69 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack, { select } from '@clack/prompts';
+import chalk from 'chalk';
+import {
+  abortIfCancelled,
+  addDotEnvSentryBuildPluginFile,
+  getPackageDotJson,
+  hasPackageInstalled,
+  installPackage,
+} from '../../utils/clack-utils';
+
+import {
+  SourceMapUploadToolConfigurationFunction,
+  SourceMapUploadToolConfigurationOptions,
+} from './types';
+
+const getCodeSnippet = (options: SourceMapUploadToolConfigurationOptions) =>
+  chalk.gray(`
+  ${chalk.greenBright(
+    'import { sentryRollupPlugin } from "@sentry/rollup-plugin";',
+  )}
+
+  export default {
+    output: {
+      ${chalk.greenBright(
+        'sourcemap: true, // Source map generation must be turned on',
+      )}
+    },
+    plugins: [
+      // Put the Sentry rollup plugin after all other plugins
+      ${chalk.greenBright(`sentryRollupPlugin({
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        org: "${options.orgSlug}",
+        project: "${options.projectSlug}",${
+        options.selfHosted ? `\n      url: "${options.url}",` : ''
+      }
+      }),`)}
+    ],
+  };
+`);
+
+export const configureRollupPlugin: SourceMapUploadToolConfigurationFunction =
+  async (options) => {
+    await installPackage({
+      packageName: '@sentry/rollup-plugin',
+      alreadyInstalled: hasPackageInstalled(
+        '@sentry/rollup-plugin',
+        await getPackageDotJson(),
+      ),
+    });
+
+    clack.log.step(
+      `Add the following code to your ${chalk.bold('rollup.config.js')} file:`,
```

Comment:
Hm, maybe just the following just so we don't make any assumptions about the users setup? But I honestly don't know.

```suggestion
      `Add the following code to your rollup config:`,
```

Reply:
Hmm yeah let's go with this. I guess since rollup can be used directly in JS, we can't even be sure they have a file... (I guess the same thing holds for Vite but I haven't seen this yet outside framework-internal builds).